test/integration/testdata/gen-no-uuid.golden

```diff
@@ -98,7 +98,7 @@ data:
       - name: SONOBUOY_K8S_VERSION
         value: v1.20.0
       image: k8s.gcr.io/conformance:v1.20.0
-      imagePullPolicy: Always
```

Comment:
There was a disconnect between the intended defaults and the _actual_ defaults. This fixes it.